src/Goerli-DssSpellCollateral.sol

```diff
@@ -0,0 +1,181 @@
+    uint256 constant TARGET_UNIV2WBTCDAI_A_MAT      = 800 * RAY / 100;
+
+    // --- Rates ---
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
+    //
+
+    function onboardNewCollaterals() internal {
+        // ----------------------------- Collateral onboarding -----------------------------
+        //  Add ______________ as a new Vault Type
+        //  Poll Link:
+
+        // DssExecLib.addNewCollateral(
+        //     CollateralOpts({
+        //         ilk:                   ,
+        //         gem:                   ,
+        //         join:                  ,
+        //         clip:                  ,
+        //         calc:                  ,
+        //         pip:                   ,
+        //         isLiquidatable:        ,
+        //         isOSM:                 ,
+        //         whitelistOSM:          ,
+        //         ilkDebtCeiling:        ,
+        //         minVaultAmount:        ,
+        //         maxLiquidationAmount:  ,
+        //         liquidationPenalty:    ,
+        //         ilkStabilityFee:       ,
+        //         startingPriceFactor:   ,
+        //         breakerTolerance:      ,
+        //         auctionDuration:       ,
+        //         permittedDrop:         ,
+        //         liquidationRatio:      ,
+        //         kprFlatReward:         ,
+        //         kprPctReward:
+        //     })
+        // );
+
+        // DssExecLib.setStairstepExponentialDecrease(
+        //     CALC_ADDR,
+        //     DURATION,
+        //     PCT_BPS
+        // );
+
+        // DssExecLib.setIlkAutoLineParameters(
+        //     ILK,
+        //     AMOUNT,
+        //     GAP,
+        //     TTL
+        // );
+
+        // ChainLog Updates
+        // Add the new flip and join to the Chainlog
+        // address constant CHAINLOG        = DssExecLib.LOG();
+        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
+        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
+        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    }
+
+    function offboardCollaterals() internal {
+
+        address spotter = DssExecLib.spotter();
+
+        // Offboard UNI-A
+        // https://vote.makerdao.com/polling/QmSfLS6V#poll-detail
+        // https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+
+        DssExecLib.setIlkLiquidationPenalty("UNI-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("UNI-A", 0);
```

Comment:
We set `tip` to `0` as explicitly requested, but `chip` is left unchanged.  From the wording in the offboarding document we make this change to avoid farming liquidation incentives.  While we're unlikely to find ourselves in this farming situation, we may not need to pay for these to offboarding at all.  I think it merits consideration that we set `chip` to `0` too.  This means, like with `TUSD`, we would need to kick these auctions when they were ready.  Note, we did not appear we adjusted `tip` or `chip` when we offboarded other collateral types.  I'm going to just put this here.  I think it's fine if governance pays for the `bark()`, but we probably should have given the `tip` (flat rate) incentive rather than the `chip` (percentage).